go.mod

```diff
@@ -44,14 +44,14 @@ require (
 	github.com/gorilla/websocket v1.5.1
 	github.com/jinzhu/copier v0.3.5
 	github.com/kubernetes-csi/csi-lib-utils v0.6.1
-	github.com/longhorn/backing-image-manager v1.5.5-rc1
+	github.com/longhorn/backing-image-manager v1.5.5-rc2
 	github.com/longhorn/backupstore v0.0.0-20240219094812-3a87ee02df77
 	github.com/longhorn/go-common-libs v0.0.0-20240411093823-b8862efb8e03
 	github.com/longhorn/go-iscsi-helper v0.0.0-20240329081212-26b6406b3b2b
 	github.com/longhorn/go-spdk-helper v0.0.0-20231113055029-9acddd184246
-	github.com/longhorn/longhorn-engine v1.5.5-0.20240319113652-9cdc0f5750a7
-	github.com/longhorn/longhorn-instance-manager v1.5.5-0.20240326065406-8d925c0e7e1e
-	github.com/longhorn/longhorn-share-manager v1.5.5-0.20240319114127-58303a9ff61f
+	github.com/longhorn/longhorn-engine v1.5.5-rc2
```

Comment:
Should [8f85db1d139340c0fcccf94837271a5c5ebc57aa](https://github.com/longhorn/longhorn-engine/commits/v1.5.x/)?